docs/DSP/GameCube_DSP_Users_Manual/GameCube_DSP_Users_Manual.tex

```diff
@@ -3603,144 +4905,199 @@ \section{Alphabetical list of extended opcodes}
-NX          &          & *  & 1000 x000 xxxx xxxx                     \\
-CLR         &          & *  & 1000 x001 xxxx xxxx                     \\
-CMP         &          & *  & 1000 0010 xxxx xxxx                     \\
-???         & UNUSED   & *  & 1000 0011 xxxx xxxx                     \\
-CLRP        &          & *  & 1000 0100 xxxx xxxx                     \\
-TSTAXH      &          & *  & 1000 011x xxxx xxxx                     \\
-M0/M2       &          &    & 1000 101x xxxx xxxx                     \\
-CLR15/SET15 &          &    & 1000 110x xxxx xxxx                     \\
-SET40/16    &          &    & 1000 111x xxxx xxxx                     \\
-            &          &    &                                         \\
-MUL         &          & *  & 1001 a000 xxxx xxxx                     \\
-ASR16       &          & *  & 1001 r001 xxxx xxxx                     \\
-MULMVZ      &          & *  & 1001 a01r xxxx xxxx                     \\
-MULAC       &          & *  & 1001 a10r xxxx xxxx                     \\
-MULMV       &          & *  & 1001 a11r xxxx xxxx                     \\
-            &          &    &                                         \\
-MULX        &          & *  & 101b a000 xxxx xxxx                     \\
-???         &          &    & 1010 r001 xxxx xxxx                     \\
-MULXMVZ     &          & *  & 101b a01r xxxx xxxx                     \\
-MULXAC      &          & *  & 101b a10r xxxx xxxx                     \\
-MULXMV      &          & *  & 101b a11r xxxx xxxx                     \\
-            &          &    &                                         \\
-MULC        &          & *  & 110s a000 xxxx xxxx                     \\
-CMP         &          & *  & 110x r001 xxxx xxxx                     \\
-MULCMVZ     &          & *  & 110s a01r xxxx xxxx                     \\
-MULCAC      &          & *  & 110s a10r xxxx xxxx                     \\
-MULCMV      &          & *  & 110s a11r xxxx xxxx                     \\
-            &          &    &                                         \\
-MADDX       &          & ** & 1110 00st xxxx xxxx                     \\
-MSUBX       &          & ** & 1110 01st xxxx xxxx                     \\
-MADDC       &          & ** & 1110 10st xxxx xxxx                     \\
-MSUBC       &          & ** & 1110 11st xxxx xxxx                     \\
-            &          &    &                                         \\
-LSL16       &          & *  & 1111 000r xxxx xxxx                     \\
-MADD        &          & *  & 1111 001s xxxx xxxx                     \\
-LSR16       &          & *  & 1111 010r xxxx xxxx                     \\
-MSUB        &          & *  & 1111 011s xxxx xxxx                     \\
-ADDPAXZ     &          & *  & 1111 10ar xxxx xxxx                     \\
-CLRL        &          & *  & 1111 110r xxxx xxxx                     \\
-MOVPZ       &          & *  & 1111 111r xxxx xxxx
+\begin{longtable}{llr}
+Instruction & Opcode & Page \\ \hline
```

Comment:
Can we make this table header repeat on a page break?

Reply:
It seems like [it's possible to do that using `\endhead`](https://tex.stackexchange.com/a/11381).  It's also [possible to use `\\*`](https://tex.stackexchange.com/a/468627) instead of `\\`, which tells latex to avoid breaking pages there (and thus to prefer breaking pages on `\OpcodeRowSkip` which will still use `\\`), which means I don't need to manually add an extra one to try and get good page wrapping (and if someone were to format it for letter paper instead of A4 paper, it'd handle it in a reasonably intelligent manner).